core/src/main/resources/org/apache/brooklyn/core/mgmt/persist/deserializingClassRenames.properties

```diff
@@ -1434,3 +1434,5 @@ brooklyn.test.entity.BlockingEntityImpl
 org.apache.brooklyn.config.ConfigInheritance$None                                : org.apache.brooklyn.config.ConfigInheritance$Legacy$None
 org.apache.brooklyn.config.ConfigInheritance$Always                              : org.apache.brooklyn.config.ConfigInheritance$Legacy$Always
 org.apache.brooklyn.config.ConfigInheritance$Merged                              : org.apache.brooklyn.config.ConfigInheritance$Legacy$Merged
+
+org.apache.brooklyn.entity.software.base.lifecycle.MachineLifecycleEffectorTasks$ProvisioningTaskState : org.apache.brooklyn.core.entity.internal.AttributesInternal$ProvisioningTaskState
```

Comment:
@neykov Agree that would be useful. Do you think we should add it in this properties file, or just on the javadoc of the class in question?

For this file, there are a bunch of entries higher up that are also "since 0.10.0" so feels like we should figure out where and put the comment on that group.

It gets a little more complicated if we want to group them with finer granularity (e.g. some downstream projects like AMP have been taking timestamped early-access releases of Brooklyn. But here we should probably just focus on the Brooklyn GA releases.

I suggest we do that separately (figuring out where the changes since 0.9.0 begin!)